languages/nodejs/repl.js

```diff
@@ -145,20 +146,23 @@ if (process.env.PRYBAR_CODE) {
     root: global,
   };
 
-  // These properties will show a warning. We can avoid them.
-  for (const prop of Object.getOwnPropertyNames(global)) {
-    if (sandbox.hasOwnProperty(prop)) {
-      continue;
-    }
-
-    sandbox[prop] = global[prop];
-  }
-
   console.log(
     "\u001b[0m\u001b[90mHint: hit control+c anytime to enter REPL.\u001b[0m"
   );
   const context = vm.createContext(sandbox);
 
+  // Adapted from https://bit.ly/2Fc4WjM
+  const globalBuiltins =
+  new Set(vm.runInNewContext('Object.getOwnPropertyNames(globalThis)'));
```

Comment:
Actually let's run this in the our context, seems more sane
```suggestion
  const globalBuiltins = vm.runInContext('Object.getOwnPropertyNames(global)', context);
```